api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/EptsCalculations.java

```diff
@@ -35,7 +34,10 @@
 import org.openmrs.module.reporting.data.person.definition.VitalStatusDataDefinition;
 import org.openmrs.util.OpenmrsUtil;
 
-/** Utility class of common base calculations */
+/**
+ * Utility class of common base calculations TODO: refactor needs to be merged with
+ * EptsCalculationUtils
```

Comment:
so why not merge them rather, do we have such a ticket @nthfloor ?

Reply:
When I added that comment I was thinking we should merge. But seeing now how those two classes have evolved I don't think we should merge anymore. 
* EptsCalculationUtils is currently a collections of helper functions 
* Whereas EptsCalculations has lookup functions for getting Obs and encounters. Kind of like a service for looking up openmrs objects (we should probably rename to be more descriptive)